### ISSUE TITLE
Windows auto-update doesn't clean up extensions directory from old extensions

### ISSUE THREAD

**Issue by jweak** [kontena/lens#6297]
Updating to the latest will keep the old bundled extensions and add the new ones because they have changed names which will cause duplicate extensions running at the same time.

Steps to reproduce: 
Update to the latest with the app update.
Go to `%AppData%`
Checkout `node_modules` and see there are still the old extensions and new extensions (survey, license etc)

**Comment by jim-docker** [comment]
This is what I see after update to 2022.9.2806.35:
![Screen Shot 2022-09-28 at 11 08 16 AM](https://user-images.githubusercontent.com/40840436/192816678-39ca3bc2-f3dc-4a4b-98d6-a7b1b239c4ce.png)

**Comment by jweak** [comment]
Interesting! Have you auto-updated from 5.x or had a fresh install after the new build system (I guess after we moved the extensions to own repos and renamed them)?

My node_modules looks like this:

<img width="1170" alt="image" src="https://user-images.githubusercontent.com/774344/192830580-e5bf0e53-f23a-4455-8259-f51bcc77e393.png">

See there's for example `lens-license` and `lens-license-extension`

**Comment by jim-docker** [comment]
Ah, I probably have done a clean install on windows since Lens 6 was released. So if I install the last 5.x version and autoupdate I should see this? And it only happens on windows?

**Comment by jim-docker** [comment]
- When I ran the 5.4 installer (overtop of my 6.1 installation) I saw the double extensions (e.g. lens-telemetry and telemetry-lens-extension).
- But when I let it update to 6.1 there was only telemetry-lens-extension.
- Also, when I ran the 6.1 installer (overtop of my 5.4 installation) there was only telemetry-lens-extension

I only see the double extensions when running an old installer overtop of a newer installation. This is for an installation for all users.
